android/src/main/java/com/swmansion/rnscreens/ScreenStackHeaderConfig.kt

```diff
@@ -22,28 +22,28 @@ import com.swmansion.rnscreens.events.HeaderAttachedEvent
 import com.swmansion.rnscreens.events.HeaderDetachedEvent
 
 class ScreenStackHeaderConfig(context: Context) : ViewGroup(context) {
-    private val mConfigSubviews = ArrayList<ScreenStackHeaderSubview>(3)
+    private val configSubviews = ArrayList<ScreenStackHeaderSubview>(3)
     val toolbar: CustomToolbar
-    var mIsHidden = false
+    var isHeaderHidden = false  // named this way to avoid conflict with platform's isHidden
```

Comment:
Do you know if there were any other conflicts in naming between internal variables and our ones? I also remember that there were a conflict of `isHidden` when I was doing a PR with Material 3 Top App Bar, but haven't spot any other conflicts - I'm just curious about that :D

Reply:
If there would be more, there would be compile-time errors I believe as it would be unambiguous definition.